apps/web/pages/stofnanir/[slug]/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import withApollo from '@island.is/web/graphql/withApollo'
+import { withLocale } from '../../../i18n/withLocale'
+import { Screen } from '@island.is/web/types'
+import { useRouter } from 'next/router'
+
+// TODO: remove this redirect route after 9.1.0 is released
+const StofnanirRedirect: Screen = () => {
+  const router = useRouter()
+
+  return (
+    <div>
+      <meta httpEquiv="refresh" content={`0;url=/s/${router.query.slug}`} />
+    </div>
+  )
+}
+
+StofnanirRedirect.getInitialProps = async ({ res, query }) => {
+  if (res) {
+    res.writeHead(302, {
+      Location: `/s/${query.slug}`,
+    })
+    res.end()
+  }
+}
```

Comment:
We have definitions in the link resolver fro english pages, do we need this redirect for them as well?

Reply:
We decided to skip them, as there is almost no content on the english pages and there Andes / Hugsmidjan don't have redirects to them.